Makefile

```diff
@@ -35,6 +44,20 @@ clean:
 	$(MAKE) -C app clean
 	$(MAKE) -C utils clean
 
+pkgclean:
+	rm -rf $(BUILDDIR)
+
+pkgprep: pkgclean
+	$(PKGBUILDROOT_CREATE_CMD)
+	tar --exclude=./pkgbuild --exclude=.git --transform 's,^\.,dattobd,' -czf $(BUILDDIR)/SOURCES/dattobd.tar.gz .
+	cp dist/dattobd.spec $(BUILDDIR)/SPECS/dattobd.spec
+
+deb: pkgprep
+	debbuild $(PKGBUILDFLAGS) $(BUILDDIR)/SPECS/dattobd.spec
+
+rpm: pkgprep
```

Comment:
Please mark the `pkg*`, `deb`, and `rpm` targets as `PHONY`.

Reply:
@Conan-Kudo done